advisories/github-reviewed/2024/04/GHSA-33c5-9fx5-fvjm/GHSA-33c5-9fx5-fvjm.json

```diff
@@ -71,6 +71,25 @@
           ]
         }
       ]
+    },
+    {
+      "package": {
+        "ecosystem": "Go",
+        "name": "k8s.io/apimachinery"
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "0.20.0-alpha.1"
```

Comment:
Is this affecting other version than the mentioned in the description ? I don't see why we need to refer to v0.20.0 here.

Reply:
I'd say this should set the fixed versions (v0.x.x) of the apimachinery module for the specific range of affected version. I don't see v0.20.0 among them.